### ISSUE TITLE
Intentional reversion of genomes-in-the-cloud version?

### ISSUE THREAD

**Issue by mbookman** [gatk-workflows/broad-prod-wgs-germline-snps-indels#5]
Commit 28633608dec41a2b24184d6363da1f15a111d151 reverted both the location of the `genomes-in-the-cloud docker` image as well as the version. Was reverting the version intentional?

```
-    "docker": "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.3.2-1510681135"
+    "docker": "broadinstitute/genomes-in-the-cloud:2.3.1-1512499786"
```

How can we get the details of the changes between these two versions of the Docker image?

**Comment by bshifaw** [comment]
Hi Matt
For the first question, we’d like to allow users to pull images from docker hub instead of gcr when possible to avoid any authentication barriers. The most up to date version currently available on dockerhub of gitc is 2.3.1-1512499786, however there is a ticket in place to update this image. 
Your second question would be better answered in the [gatk forum](https://gatkforums.broadinstitute.org/gatk/categories/ask-the-team) and would be an answer of interest for other gatk users.

**Comment by mbookman** [comment]
Asked!

https://gatkforums.broadinstitute.org/gatk/discussion/11815/dockerfile-history-for-genomes-in-the-cloud/p1
